index.js

```diff
@@ -17,7 +17,9 @@ try {
     core.exportVariable('SSH_AUTH_SOCK', authSock);
 
     console.log("Adding private key to agent");
-    child_process.execSync('ssh-add -', { input: core.getInput('ssh-private-key') });
+    core.getInput('ssh-private-key').split(/(?=-----BEGIN)/).forEach(function(key) {
+        child_process.execSync('ssh-add -', { input: key });
```

Comment:
It'd be nice to calculate a fingerprint and log it so that people would understand which key gets loaded. It's good for troubleshooting as there's no other ways to see what's going on.

Use something like this maybe: https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#verifying-private-keys.